content/build/hydrobuild.md

```diff
@@ -54,58 +54,103 @@ Docker Desktop 4.23.0 and later versions ship with a Hydrobuild-compatible
 Buildx binary. Alternatively, you can download and install the binary manually
 from [this repository](https://github.com/docker/buildx-desktop).
 
-## Connecting to Hydrobuild
+## Connect to Hydrobuild
 
-To start building with Hydrobuild, you must create a new builder using the
-`docker buildx create` command. The builder is connected to Hydrobuild through
-an endpoint that you specify. The endpoint represents a single, isolated
-builder. Builder endpoints have the following format:
+To start using Hydrobuild, you must first add the builder's endpoint using the
+`docker buildx create` with the `cloud` driver.
 
-cloud://<org>/default
+{{< tabs group="ui" >}}
+{{< tab name="CLI" >}}
 
-`<org>` is the Docker organization that the builder is provisioned for.
+1. Sign in to your Docker account
 
-The builders have native support for the `linux/amd64` and `linux/arm64`
-architectures by default. This gives you a high-performance build cluster for
-building multi-platform images natively.
+   ```console
+   $ docker login
+   ```
 
-You can omit the `cloud://` protocol prefix from the endpoint when you create a
-builder using the `cloud` driver.
+2. Add the Hydrobuild endpoint.
 
-### Create a builder
+   ```console
+   $ docker buildx create --driver cloud <ORG>/default
+   ```
 
-1. Sign in to your Docker ID using the Docker Desktop UI or the `docker login`
-   command.
+   Replace `<ORG>` with the Docker Hub namespace of your Docker organization.
 
-2. Create a builder that uses the `cloud` driver.
+This creates a builder named `cloud-<org>-default`.
 
-   ```console
-   $ docker buildx create --driver cloud <org>/default
-   ```
+{{< /tab >}}
+{{< tab name="GUI" >}}
 
-   Replace `<org>` with the Docker organization.
+Enable the [Builds view](../desktop/use-desktop/builds.md) in Docker Desktop
+and complete the following steps:
 
-   This creates a builder named `cloud-<org>-default`.
+1. Sign in to your Docker account using the **Sign in** button in Docker Desktop.
+
+2. Open the Docker Desktop settings and navigate to the **Builders** tab.
+
+3. Under **Available builders**, select **Create builder**.
+
+   ![Create builder GUI screenshot](./images/create-builder-gui.webp)
+
+{{< /tab >}}
+{{< /tabs >}}
+
+The builder has native support for the `linux/amd64` and `linux/arm64`
+architectures. This gives you a high-performance build cluster for building
+multi-platform images natively.
 
 ## Use Hydrobuild from the CLI
 
-To run a build using Hydrobuild, invoke a build command and specify the
-name of the builder using the `--builder` flag.
+To run a one-time build using Hydrobuild, invoke a build command and specify
```

Comment:
"To run a test build..." maybe?

Reply:
I think I can omit this bit entirely to make it simpler